src/plugins/trace/dll/DataModel/QuicStream.cs

```diff
@@ -191,6 +191,15 @@ internal void AddEvent(QuicEvent evt, QuicState state)
                                 Timings.UpdateToState(QuicStreamState.ProcessRecv, Connection.LastScheduleStateTimeStamp, true);
                             }
                             Timings.UpdateToState(QuicStreamState.Decrypt, Timings.RecvPacket.PacketDecrypt);
+
+                            if (Timings.RecvPacket.PacketDecryptComplete == Timestamp.Zero)
```

Comment:
we need to do this for encryption as well?

Reply:
No. It's not quite the same.